docs/libcurl/opts/CURLOPT_WRITEFUNCTION.3

```diff
@@ -63,8 +63,11 @@ that amount differs from the amount passed to your callback function, it
 signals an error condition to the library. This causes the transfer to get
 aborted and the libcurl function used returns \fICURLE_WRITE_ERROR\fP.
 
-You can also abort the transfer by returning CURL_WRITEFUNC_ERROR (added in
-7.87.0), which makes \fICURLE_WRITE_ERROR\fP get returned.
+You can stop the transfer by returning CURL_WRITEFUNC_ERROR (added in 7.87.0),
+which makes \fICURLE_WRITE_ERROR\fP get returned.
+
+You can abort the transfer by returning CURL_WRITEFUNC_ABORT (added in 8.5.0),
```

Comment:
i think users may find this confusing. if stopping and aborting are the same thing except for the error code then i would refer to both of them one way or the other